contracts/src/systems/inner_hex/interface.cairo

```diff
@@ -0,0 +1,24 @@
+use dojo::world::IWorldDispatcher;
+
+use eternum::models::buildings::BuildingCategory;
+use eternum::models::position::{Coord, Position, Direction};
+
+#[starknet::interface]
+trait IRealmMap<TContractState> {
+    fn build_building(
```

Comment:
I avoided calling this 'Realm' incase we want to have banks that have similar properties